src/components/Training/course/CourseWrapper.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+import is from 'styled-is'
+
+import Layout from '../../../components/layout'
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background-color: ${props => props.theme.colors.white};
+  position: fixed;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  z-index: ${props => props.theme.zIndexes.modal};
+  transition: transform ${props => props.theme.animations.normal},
+    opacity ${props => props.theme.animations.normal};
+  transform: scale(0.7);
+  visibility: hidden;
+
+  ${is('visible')`
```

Comment:
I wanted to ask about this but forgot: I can't imagine a scenario where the modal has `visible = false`, since it will only render for provided content. In that case, can you try to remove lines 16-24?